templates/web/northumberland/footer_extra.html

```diff
@@ -18,7 +18,7 @@ <h3>Social Media</h3>
             <div class="footer-main--links col-n2">
                 <h3>Menu</h3>
                 <a href="https://www.northumberland.gov.uk/About/Contact/contacts.aspx#contactus">Contact Us</a>
-                <a href="https://www.northumberland.gov.uk/About/Contact/Information.aspx#privacynotices" rel="nofollow">Privacy Policy</a>
+                <a href="https://www.northumberland.gov.uk/NorthumberlandCountyCouncil/media/About-the-Council/information%20governance/Privacy-notice-Fix-My-Street.pdf" rel="nofollow">Privacy Policy</a>
```

Comment:
Might be better to use the `privacy_policy_url` here, rather than duplicating the URL.

```suggestion
                <a href="[% c.cobrand.privacy_policy_url %]" rel="nofollow">Privacy Policy</a>
```